tox.ini

```diff
@@ -95,7 +94,7 @@ commands =
     python --version
 
     ; Install PyDoctor here so it DOESNT overwrite Twisted
-    py27-{alldeps}: pip install --no-deps epydoc pydoctor
+    py27-{alldeps,tests}: pip install --no-deps epydoc pydoctor
```

Comment:
It got a tests because it had a tests before; it's now a 'alldeps' alias, but an earlier merge broke it :)